addons/mail/models/mail_activity.py

```diff
@@ -469,6 +470,15 @@ def name_get(self):
     # Business Methods
     # ------------------------------------------------------
 
+    def _render_notify_header(self):
```

Comment:
As it is used only once, do we need a method for that ?

Reply:
I have removed the method and inlined the content. It is indeed strange to have a method only used once.